test/README.md

```diff
@@ -164,17 +173,21 @@ At the end of each run, test results summary will be printed:
 
 ```
     TEST TARGETS SUMMARY
-    ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
     PASSED test targets:
         cmdLineTester_javaAssertions_0
         cmdLineTester_LazyClassLoading_0
         ...
+    DISABLED test targets:
+	    cmdLineTester_pltest_tty_extended_0
+	    cmdLineTester_pltest_numcpus_notBound_0
+        ...
     FAILED test targets:
         TestAttachAPIEnabling_SE80_0
         TestFileLocking_SE80_0
 
-    TOTAL: 84   EXECUTED: 84   PASSED: 82   FAILED: 2   SKIPPED: 0
-    ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    TOTAL: 91   EXECUTED: 84   PASSED: 82   FAILED: 2   SKIPPED: 0   DISABLED: 7
+    +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
```

Comment:
- I feel it may make more sense to have it displayed in order of DISABLED, PASSED and FAILED
- And have DISABLED before SKIPPED (SKIPPED does not count into TOTAL, but DISABLED does)

Reply:
agree